packages/location-field/src/LocationField.story.js

```diff
@@ -73,73 +72,69 @@ const userLocationsAndRecentPlaces = [
   }
 ];
 
-storiesOf("LocationField", module)
-  .addDecorator(withA11y)
-  .addDecorator(withInfo)
-  .add("LocationField in desktop context", () => (
+function LocationFieldStoryWrapper(props) {
```

Comment:
I'm not sure I love the use of this wrapper function for use in the LocationField story. I feel like one of the benefits of Storybook is showing the use of the components in a relatively unobstructed way. Wrapping them in this function obscures which props are needed in each scenario.